src/huggingface_hub/_login.py

```diff
@@ -186,7 +186,7 @@ def interpreter_login(new_session: bool = True, write_permission: bool = False)
     print("    To login, `huggingface_hub` requires a token generated from https://huggingface.co/settings/tokens .")
     if os.name == "nt":
         print("Token can be pasted using 'Right-Click'.")
-    token = getpass("Token: ")
+    token = getpass("Token (will not echo): ")
```

Comment:
Hi @vtrenton, thanks for opening the PR. If it happened to you, it will probably happened to someone else as well. I would make the message even more informative. What do you think of this? 

```suggestion
    token = getpass("Enter your token (input will not be visible): ")
```